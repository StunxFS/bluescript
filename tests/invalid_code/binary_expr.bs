fn main() {
    _ = 1 && "";
    _ = nil || 1;
}