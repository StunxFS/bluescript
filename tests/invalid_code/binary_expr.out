tests/invalid_code/binary_expr.bs:2:14: error: operator `&&` is not defined for type `string`
   └ note: operator `&&` is only defined for type `bool`
tests/invalid_code/binary_expr.bs:3:16: error: operator `||` is not defined for type `int`
   └ note: operator `||` is only defined for type `bool`
